testing_and_setup/compass/general.config.landice

```diff
@@ -25,11 +25,9 @@ forward = FULL_PATH_TO_FORWARD_TEMPLATE_STREAMS
 # anywhere on the machine.
 [executables]
 model = FULL_PATH_TO_MODEL
-mesh_converter = FULL_PATH_TO_MESH_CONVERTER
-cell_culler = FULL_PATH_TO_CELL_CULLER
-metis = FULL_PATH_TO_METIS
-grid_to_li_grid = FULL_PATH_TO_create_landice_grid_from_generic_MPAS_grid.py
-periodic_hex = FULL_PATH_TO_PERIODIC_HEX
+grid_to_li_grid = FULL_PATH_TO/MPAS-Tools/landice/mesh_tools_li/create_landice_grid_from_generic_MPAS_grid.py
+define_cull_mask = FULL_PATH_TO/MPAS-Tools/landice/mesh_tools_li/define_cullMask.py
+interpolate_to_mpasli_grid = FULL_PATH_TO/MPAS-Tools/landice/mesh_tools_li/interpolate_to_mpasli_grid.py
```

Comment:
Presumably these 3 scripts should just be in the path when you load the `compass` environment.  That's basically the reason for the reorganization.  Is the plan to remove these 3 lines once the `compass` environment gets updated after https://github.com/MPAS-Dev/MPAS-Tools/pull/260 gets merged?

Reply:
Yes, I only started updating the LI COMPASS - there will be a whole raft of additional commits here once I have merged the MPAS-Tools updates.

Reply:
Perfect! Just wanted to be sure.